vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/bitflyer/post_claim/post_claim_bitflyer.cc

```diff
@@ -0,0 +1,145 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/endpoint/bitflyer/post_claim/post_claim_bitflyer.h"
+
+#include <utility>
+
+#include "base/json/json_writer.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/common/request_util.h"
+#include "bat/ledger/internal/ledger_impl.h"
+#include "bat/ledger/mojom_structs.h"
+#include "net/http/http_status_code.h"
+
+using std::placeholders::_1;
+
+namespace {
+
+const char kDevelopment[] = "https://grant.rewards.brave.software";
```

Comment:
nit: These endpoints exist in `rewards_server_util` | `promotion_util` - worth consolidating in a common file?

Reply:
@emerick Should this particular endpoint be moved into the "promotion" folder?

Reply:
Yeah, that probably makes sense.

Reply:
This is done.